lnwallet/log.go

```diff
@@ -34,6 +35,7 @@ func UseLogger(logger btclog.Logger) {
 	btcwallet.UseLogger(logger)
 	wtxmgr.UseLogger(logger)
 	chain.UseLogger(logger)
+	chainfee.UseLogger(logger)
```

Comment:
This will cause the default `chainfee` package's logger `CFEE` to be overwritten. So it will continue to log as `LNWL`. Is this on purpose?
Otherwise we might want to create a new logger in the root package's `log.go` for the `chainfee` package.

Reply:
As is, yes so it remains under the same logging namespace. However external packages can have it be a distinct logger if they wish.